templates/zerver/api/main.html

```diff
@@ -1,35 +0,0 @@
-{% extends "zerver/portico-help.html" %}
-
-{# API information page #}
-
-{% block portico_content %}
-<div class="app help api-docs terms-page inline-block">
-    <div class="sidebar">
-        <div class="content">
-            <h1 class="no-arrow"><a href="/api/" class="no-underline">Home</a></h1>
-            {{ render_markdown_path("zerver/api/sidebar.md", api_uri_context) }}
-            <h1 class="home-link"><a href="/" class="no-underline">Back to Home</a></h1>
-        </div>
-    </div>
-
-    <svg height="32px" class="hamburger" style="enable-background:new 0 0 32 32;" version="1.1" viewBox="0 0 32 32" width="32px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-        <path d="M4,10h24c1.104,0,2-0.896,2-2s-0.896-2-2-2H4C2.896,6,2,6.896,2,8S2.896,10,4,10z M28,14H4c-1.104,0-2,0.896-2,2  s0.896,2,2,2h24c1.104,0,2-0.896,2-2S29.104,14,28,14z M28,22H4c-1.104,0-2,0.896-2,2s0.896,2,2,2h24c1.104,0,2-0.896,2-2  S29.104,22,28,22z"></path>
-    </svg>
```

Comment:
I'm getting local linter errors from this thing, but I don't know what it is, what it does, how to look at it, or otherwise how to mess with it to fix the linting errors. 

The linter is complaining about having styling inline instead of in a CSS file.

Reply:
Nevermind. There was an exception for this that needed to be updated.